file-formats/tabl/type/zif_aff_indx_v1.intf.abap~

```diff
@@ -0,0 +1,42 @@
+INTERFACE zif_aff_indx_v1
+  PUBLIC.
```

Comment:
bad filename, should end with `.abap` instead of `.abap~`

Reply:
I agree. I guess this file should be deleted, isn't it? There is already a file `zif_aff_indx_v1.intf.abap.

Also file `.zif_aff_indx_v1.intf.abap.un~` should be removed